Content.Client/VoiceMask/VoiceMaskNameChangeWindow.xaml

```diff
@@ -2,15 +2,20 @@
                xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
                Title="{Loc 'voice-mask-name-change-window'}"
                MinSize="5 30">
-    <BoxContainer Orientation="Vertical" Margin="5">
-        <Label Text="{Loc 'voice-mask-name-change-info'}" />
-        <BoxContainer Orientation="Horizontal" Margin="5">
-            <LineEdit Name="NameSelector" HorizontalExpand="True" />
-            <Button Name="NameSelectorSet" Text="{Loc 'voice-mask-name-change-set'}" />
-        </BoxContainer>
-        <BoxContainer Orientation="Horizontal" Margin="5">
-            <Label Text="{Loc 'voice-mask-name-change-speech-style'}" />
-            <OptionButton Name="SpeechVerbSelector" /> <!-- Populated in LoadVerbs -->
-        </BoxContainer>
+    <BoxContainer Orientation="Vertical" Margin="7">
+        <Label Text="{Loc 'voice-mask-name-change-info'}" Margin="0 0 0 10" />
+        <GridContainer Columns="2">
+            <GridContainer Rows="3" HorizontalExpand="True">
+                <LineEdit Name="NameSelector" MinWidth="175" HorizontalExpand="True" Margin="0 0 10 10" />
+                <Label Text="{Loc 'voice-mask-name-change-speech-style'}" Margin="0 0 0 5" />
+                <Label Text="{Loc 'voice-mask-name-change-speech-sound'}" />
+            </GridContainer>
+            <GridContainer Rows="3">
+                <Button Name="NameSelectorSet" Text="{Loc 'voice-mask-name-change-set'}" Margin="0 0 0 10" />
+                <OptionButton Name="SpeechVerbSelector" /> <!-- Populated in AddVerbsAndSounds -->
+                <OptionButton Name="SpeechSoundSelector" /> <!-- Populated in AddVerbsAndSounds -->
+            </GridContainer>
+        </GridContainer>
+
```

Comment:
I had some fun playing about and made some adjustments to move the label to below the text input and make it slightly darker/smaller.

I also removed the extra GridContainer as you're meant to just list them all off (this means you won't have to do the weird hack where you have to copy the margin between the subgrids as this is what the main grid is built to do).

```suggestion
        <GridContainer Columns="2">
            <LineEdit Name="NameSelector" HorizontalExpand="True" />
            <Button Name="NameSelectorSet" Text="{Loc 'voice-mask-name-change-set'}" />
            <Label StyleClasses="LabelSubText" Text="{Loc 'voice-mask-name-change-info'}" />
            <Control /> <!-- GridContainer needs a padding element so it goes to the next row -->
            <Label Text="{Loc 'voice-mask-name-change-speech-style'}" />
            <OptionButton Name="SpeechVerbSelector" />
            <Label Text="{Loc 'voice-mask-name-change-speech-sound'}" />
            <OptionButton Name="SpeechSoundSelector" />
        </GridContainer>
```

![image](https://github.com/user-attachments/assets/a8683798-d137-4b7b-bf48-b5333096c6f1)

If you don't like the padding introduced by the Label, you can use this to trick the grid into ignoring it's width while still being able to render fine.

```xaml
            <Label StyleClasses="LabelSubText" Text="{Loc 'voice-mask-name-change-info'}" MaxWidth="1"
                   HorizontalAlignment="Left" />
```